railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -283,10 +283,6 @@ def devcontainer
   end
 
   module Generators
-    # We need to store the RAILS_DEV_PATH in a constant, otherwise the path
-    # can change in Ruby 1.8.7 when we FileUtils.cd.
```

Comment:
Then I guess the constant is only to ensure consistency & avoid a lot of `File.expand_path` calls. Removing comment okay?

Reply:
I am not sure how to interpret the comment, it could mean that the path can change since Ruby 1.8.7. It is still true today that `FileUtils.cd` changes the working directory (which would affect `File.expand_path` among other things). I don't know what pre-1.8.7 behavior was. Maybe a conservative change could be to only remove "in Ruby 1.8.7" from the comment.

Reply:
updated.